streaming/java/streaming-state/src/main/java/org/ray/streaming/state/serde/KeyMapStoreSerialization.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ray.streaming.state.serde;
+
+/**
+ * Key Map Serialization and Deserialization.
+ */
+public interface KeyMapStoreSerialization<K, S, T> {
+
+  byte[] serKey(K key);
```

Comment:
I thin all abbr like `ser/serde/deser` should be avoided. Name like this spreads many places.